mycroft/skills/intent_service.py

```diff
@@ -103,12 +106,25 @@ def __init__(self, bus):
         # Converse method
         self.bus.on('mycroft.speech.recognition.unknown', self.reset_converse)
         self.bus.on('mycroft.skills.loaded', self.update_skill_name_dict)
+        self.bus.on('mycroft.skills.list', self.update_skill_list)
+
 
         def add_active_skill_handler(message):
-            self.add_active_skill(message.data['skill_id'])
+            category = 'undefined'
+            if message.data.get("skill_cat", None) is not None:
```

Comment:
None is the default value returned by get() if no match is found.  you don't need to specify it.

Reply:
done